menpofit/sdm/fitter.py

```diff
@@ -77,9 +77,8 @@ def _train(self, images, group=None, bounding_box_group=None,
 
         # In the case where group is None, we need to get the only key so that
         # we can attach landmarks below and not get a complaint about using None
-        first_image = first_batch[0]
```

Comment:
Yes you're definitely right, that's not copying! Because of the line:

```
for k, image_batch in enumerate(chain([first_batch], image_batches)):
```

the landmarks attached to `image_batch` are not present in `first_batch`